invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/SearchMain/SearchFacets/SearchFacets.js

```diff
@@ -0,0 +1,58 @@
+// This file is part of InvenioRDM
```

Comment:
Could you crosscheck with @lnielsen or @slint if this line is needed/affects something? is not there in many of the modules...